protocol/src/main/java/org/apache/drill/exec/proto/UserBitShared.java

```diff
@@ -133,6 +133,10 @@ private RpcChannel(int index, int value) {
      * <code>PHYSICAL = 3;</code>
      */
     PHYSICAL(2, 3),
+    /**
+     * <code>EXECUTIONAL = 4;</code>
+     */
+    EXECUTIONAL(3, 4),
```

Comment:
Unlike 'Logical' and 'Physical',  EXECUTIONAL seems odd since it is not a real word :)  If you want to convey that this plan is directly executable (it was already fully planned before), should it just be EXECUTABLE instead ?